jdbc/spring-cloud-starter-stream-source-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,3 +1,4 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.source.JdbcSourceProperties, \
-  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceProperties
```

Comment:
Yeah... I wonder if that is really a result of an artifact in your `mavenLocal`...